Libs/MRML/DisplayableManager/vtkMRMLThreeDViewInteractorStyle.cxx

```diff
@@ -61,6 +64,29 @@ void vtkMRMLThreeDViewInteractorStyle::PrintSelf(ostream& os, vtkIndent indent)
   this->Superclass::PrintSelf(os,indent);
 }
 
+//----------------------------------------------------------------------------
+void vtkMRMLThreeDViewInteractorStyle::SetDisplayableManagers(vtkMRMLDisplayableManagerGroup* displayableManagers)
+{
+  if (displayableManagers == this->DisplayableManagers)
+    {
+    return;
+    }
+
+  this->Superclass::SetDisplayableManagers(displayableManagers);
+
+  // Observe ActiveCameraChangedEvent
+  vtkMRMLAbstractDisplayableManager* cameraDisplayableManager =
+      this->DisplayableManagers->GetDisplayableManagerByClassName("vtkMRMLCameraDisplayableManager");
+  if (cameraDisplayableManager)
+    {
+    // \tbd active camera should be set to view node instead and only observing
```

Comment:
I would not say it is a TODO. It would make sense to describe why this is here (to detect when the camera displayable manager sets the camera) and maybe make a note on that a simpler alternative would be to directly store a reference to the camera in the view node. However, if we really want to simplify how the camera is mapped to the view (which would make sense) then we should rather add an issue to the bugtracker.

Reply:
Agreed. I will also update the comment at https://github.com/Slicer/Slicer/blob/9f95ec7deb1045167c59644a7634ecde5b60ac8a/Libs/MRML/DisplayableManager/vtkMRMLViewDisplayableManager.cxx#L652-L655 and add an entry in the tracker.